mirascope/chat/types.py

```diff
@@ -127,12 +180,11 @@ def content(self) -> Optional[str]:
     def tool_calls(self) -> Optional[list[ChoiceDeltaToolCall]]:
         """Returns the partial tool calls for the 0th choice message.
 
-        The first and last `list[ChoiceDeltaToolCall]` will be None indicating start
-        and end of stream respectively. The next `list[ChoiceDeltaToolCall]` will
-        contain the name of the tool and index and subsequent
-        `list[ChoiceDeltaToolCall]`s will contain the arguments which will be strings
-        that need to be concatenated with future `list[ChoiceDeltaToolCall]`s to form a
-        complete JSON tool calls.
+        The first `list[ChoiceDeltaToolCall]` will contain the name of the tool and
```

Comment:
@brenkao we need to confirm this different found here

Reply:
is there supposed to be a link